docs/modules/ROOT/pages/custom-config.adoc

```diff
@@ -144,3 +145,7 @@ Some Hazelcast features require the management of additional K8s resources (e.g.
 === Networking configuration
 
 The Hazelcast Platform Operator communicates with the Hazelcast cluster, requiring a specific networking configuration. Reconfiguring it using Custom Config might break communication between the Operator and the cluster. To avoid such cases, the Operator will always override the networking section of the Custom Config. If you require any specific networking setup, please use Hazelcast CR.
+
+=== Security configuration
+
+When configuring `security` with enabled `client-authentication`, you must provide a user with `all` permissions and a simple authentication method. The client and the operator will use this user to connect to and manage the cluster.
```

Comment:
@SeriyBg could you update existing configmap and secret samples based on this requirements?

Reply:
We don't have the sample of custom configs in the operator codebase. In the docs, however, there are two examples with the security, but it does not enable client auth.

Reply:
> In the docs, however, there are two examples with the security, but it does not enable client auth.

I think we should modify them or at least we should another provide sample YAML that contains requires stuff directly here

Reply:
Done